bundles.md

```diff
@@ -285,3 +285,47 @@ New list of bitstreams:
   ]
 }
 ```
+
+## Primary bitstream
+
+The primary bitstream is exposed as a link, and can be modified
+
+**/api/core/bundles/<:uuid>/primaryBitstream**
+
+### Retrieve Primary bitstream
+
+The Primary bitstream can be retrieved using a GET, this may result in `null`
+
+**GET /api/core/bundles/<:uuid>/primaryBitstream**
+
+### Create Primary bitstream
+
+If the Primary bitstream doesn't exist, it can be created using a POST
+
+**POST /api/core/bundles/<:uuid>/primaryBitstream**
+
+This should use `Content-Type:text/uri-list` with a single URI value which represents the new Bitstream to mark as primary.
+
+```
+curl -i -X POST "https://demo7.dspace.org/api/core/bundles/[uuid]/primaryBitstream" -H "Content-type:text/uri-list" -d "https://demo7.dspace.org/api/core/bitstreams/[uuid]"
+```
+
+### Update Primary bitstream
+
+If the Primary bitstream already exists, it can be changed using a PUT
+
+**PUT /api/core/bundles/<:uuid>/primaryBitstream**
+
+This should use `Content-Type:text/uri-list` with a single URI value which represents the new Bitstream to mark as primary.
+
+```
+curl -i -X PUT "https://demo7.dspace.org/api/core/bundles/[uuid]/primaryBitstream" -H "Content-type:text/uri-list" -d "https://demo7.dspace.org/api/core/bitstreams/[uuid]"
+```
```

Comment:
it will be nice if you can add explicitly the return code (auth/authz error, 200 success, 404, 422)